package.json

```diff
@@ -27,6 +27,8 @@
     "webpack.config.js"
   ],
   "scripts": {
+    "lint": "eslint ./",
+    "lint-fix": "eslint ./ --fix",
     "test": "npm run utest-coverage && npm run itest",
```

Comment:
Linting should be part of the `npm test` script so that it is executed by Travis CI.